src/ServicePointsModal/ServicePointsModal.js

```diff
@@ -13,17 +17,23 @@ class ServicePointsModal extends React.Component {
   };
 
   setCurrentServicePoint(servicePoint) {
-    const { stripes: { store }, onClose } = this.props;
-    setCurServicePoint(store, servicePoint);
+    const { stripes, onClose } = this.props;
+
+    // persist to storage
+    updateUser(stripes.store, { curServicePoint: servicePoint });
+
+    // rerender root
+    stripes.updateUser({ curServicePoint: servicePoint });
```

Comment:
It looks like `updateUser` above also updates redux store:

https://github.com/folio-org/stripes-core/pull/1243/files#diff-c9d695b1d57404dfed01196c8cf479c0e4cd48583f6c78e7db23402582846864R644

I think we can remove this right?